ros_buildfarm/templates/release/rpm/binarypkg_task.Dockerfile.em

```diff
@@ -2,16 +2,16 @@
 package_manager = 'dnf'
 python3_pkgversion = '3'
 
-if os_name in ['centos', 'rhel'] and os_code_name.isnumeric() and int(os_code_name) < 8:
+if os_name in ['rhel'] and os_code_name.isnumeric() and int(os_code_name) < 8:
```

Comment:
nitpick: seems weird to find an `in [...]` with just one value while the code seems to use `==`. Not sure if you want to leave the list there for maintainability reasons.

Reply:
1ff0d6d